### PR TITLE
LPS-102110 SF

### PR DESCRIPTION
**Pull request by jbalsas**

### CODE REVIEW

**Comment by jbalsas** [comment]
ci:test:sf

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:test

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/scheduler.js [hunk 1]
```diff
@@ -681,8 +681,11 @@ AUI.add(
 
 					Scheduler.superclass.renderUI.apply(this, arguments);
 
-					instance.navDateNode.replaceClass('hidden-xs', 'hidden');
```
Comment (jbalsas) [inline modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/scheduler.js]: We should test this, I'm not sure the JS API works this way...

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/clay/_utilities.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.hide {
+	@extend .d-none;
```
Comment (jbalsas) [inline modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/clay/_utilities.scss]: Is this no longer needed @marcoscv-work, @eduardoallegrini?

**Comment by jbalsas** [comment]
Hey @john-co, could you please review these test results? Trying to start pushing this out now that SP1 is done

**Comment by john-co** [comment]
Just started reviewing :)

**Comment by john-co** [comment]
ci:test:bundle

**Comment by john-co** [comment]
Hi @jbalsas,

Related functional test failure:

Navigating to Add forms and clicking anywhere on page displays `"Are you sure you want to deactivate this language?"` modal.

Forms_deactivateThisLanguage

js console log error:
Selection_140

Backend test failures look unrelated.

**Comment by jbalsas** [comment]
Thanks @john-co! Managed to reproduce this. It happens because we now have 

```
.hide { display: none; }
```

rather than

```
.hide { display: none !important; }
```

I think we probably want the latter since we're not changing all `.hide` usages

**Comment by jbalsas** [comment]
https://github.com/eduardoallegrini/liferay-portal/pull/26

**Pull request closed without merging**